src/Diagnostics/FxCop/System.Runtime.Analyzers/Test/Design/DefineAccessorsForAttributeArgumentsTests.Fixer.cs

```diff
@@ -113,7 +113,7 @@ public InternalGetterTestAttribute(string name)
     public string Name
     {
         internal get { return m_name; }
-        set { m_name = value; }
+        internal set { m_name = value; }
```

Comment:
[minor] This code is not valid C# (it produces another error CS0274 'Cannot specify accessibility modifiers for both accessors of the property or indexer'.) Consider adding a scenario that is valid C# and only produces CA1019.